lib/mbr.mli

```diff
@@ -79,11 +79,11 @@ type t = private {
   seconds : int;
   minutes : int;
   hours : int;
-  disk_signature : int32;
+  disk_signature : int32 option;
   partitions : Partition.t list;
 }
 
-val make : Partition.t list -> (t, string) result
+val make : Partition.t list -> int32 -> (t, string) result
```

Comment:
The signature for functions with optional arguments look a bit different. This signature is for a function with a mandatory unlabelled argument.

See https://v2.ocaml.org/releases/5.0/htmlman/lablexamples.html#s:optional-arguments

Reply:
Okay I think I get it, you mean I did not have to create a `signature`, I never knew what that was, after doing some research, I wish this will do ```val make : ?disk_signature:int32 -> 'a list -> 'a``
`